.github/workflows/builds.yml

```diff
@@ -58,10 +58,10 @@ jobs:
         run: nuget pack Source\Mosa.Packages\Mosa.DeviceSystem.nuspec -OutputDirectory bin\nupkg -Version ${{ env.BUILD_VERSION }}
       - name: Create Package - Mosa.Runtime
         run: nuget pack Source\Mosa.Packages\Mosa.Runtime.nuspec -OutputDirectory bin\nupkg -Version ${{ env.BUILD_VERSION }}
+      - name: Create Package - Mosa.Kernel
+        run: nuget pack Source\Mosa.Packages\Mosa.Kernel.nuspec -OutputDirectory bin\nupkg -Version ${{ env.BUILD_VERSION }}
       - name: Create Package - Mosa.Platform.x86
         run: nuget pack Source\Mosa.Packages\Mosa.Platform.x86.nuspec -OutputDirectory bin\nupkg -Version ${{ env.BUILD_VERSION }}
-      - name: Create Package - Mosa.Kernel.x86
```

Comment:
This package is still used by the starter template.
Also the x64 kernel package step is missing